### PR TITLE
Spelling Corrections

### PR DESCRIPTION
**Pull request by djyasin**
Working on running spellcheck on everything ahead of merging the shellcheck/code check CI addition.

##### SUMMARY

Fixing existing spelling errors ahead of merging and finalizing #13221

##### ISSUE TYPE

 - Bug, Docs Fix or other nominal change

##### COMPONENT NAME

 - Other

##### AWX VERSION

```
21.10.3
```

### CODE REVIEW

**Review by gamuniz — APPROVED** [review]
LGTM

**Review by TheRealHaoLiu — APPROVED** [review]

**Comment by djyasin** [comment]
I don't think these changes actually will impact any functionality, but I wanted to make sure it still went through the "Needs Test" column. I will merge once it has QE approval.

**Review by jay-steurer — APPROVED** [review]

**Pull request merged**
